jenkins-builds/production.sh

```diff
@@ -7,18 +7,19 @@ scriptname=$(basename $0)
 
 usage() {
     echo "Usage: $0 [OPTIONS] <list-of-ebfiles>
-    -a,--arch     Architecture (gpu or mc)      (mandatory: Dom and Piz Daint only)
-    -f,--force    Force build of given package  (optional: double quotes for a list)
+    -a,--arch     Architecture (gpu or mc)           (mandatory: Dom and Piz Daint only)
+    -f,--force    Force build of given package       (optional: double quotes for a list)
     -h,--help     Help message
-    -l,--list     Production list file          (mandatory: EasyBuild production list)
-    -p,--prefix   EasyBuild prefix folder       (mandatory: installation folder)
-    -x,--xalt.    [yes|no] update XALT database (optional, default is yes)
+    -l,--list     Production list file               (mandatory: EasyBuild production list)
+    -p,--prefix   EasyBuild prefix folder            (mandatory: installation folder)
+    -u,--unuse    Module unuse colon separated PATH  (optional: default is null)
```

Comment:
I have implemented this flag to treat every system in the same way within the Jenkins project `RegressionEB`: this approach has the advantage to select explicitly the path to be unused, avoiding the need to have it hard coded for Piz Daint only within the script `production.sh`.
The flags `-u $PATH -x no` are used together for Dom and Piz Daint only.